packages/react-native-bridge/android/react-native-bridge/build.gradle

```diff
@@ -81,43 +59,31 @@ dependencies {
 
     testImplementation "junit:junit:4.13"
 
+    def rnVersion = readReactNativeVersion('../../../../package.json', 'devDependencies')
+    println "react-native version for react-native-bridge: $rnVersion"
+    def packageJson = '../../../react-native-editor/package.json'
+
+    implementation "com.facebook.react:react-native:$rnVersion"
+    implementation "org.wordpress-mobile:react-native-svg:${extractPackageVersion(packageJson, 'react-native-svg', 'dependencies')}"
```

Comment:
Note that the dependencies that now use the original repository are using the group ID `org.wordpress-mobile`, which is where they are published via https://github.com/wordpress-mobile/react-native-libraries-publisher repository.